steps/monitoring-apps-logs/1.0.0/step.yml

```diff
@@ -0,0 +1,48 @@
+title: Mobile apps logs
+summary: |
+  Check if your Android app kept any logs in your release build
+description: "Check if an Android application contain any logs in the release app\nIf
+  yes, this step will fail and generate a fail where you can see the count \n\n###
+  Troubleshooting\n\n- Be aware, you have to launch this step after [apps-decompiler](https://github.com/imranMnts/bitrise-step-apps-decompiler)
+  to have informations from your APK\n"
+website: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions
+source_code_url: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions
+support_url: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions/issues
+published_at: 2021-03-24T14:10:22.874866+01:00
+source:
+  git: https://github.com/imranMnts/bitrise-step-monitoring-apps-logs.git
+  commit: 0791e5e935687943fcda2a3766621d0270fec15c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- react-native
+- flutter
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: false
+inputs:
+- check_android: "yes"
+  opts:
+    description: To check or not android permissions
+    is_expand: true
+    is_required: true
+    summary: Do you want check android permissions?
+    title: Check Android
```

Comment:
It should be rather called "Check permissions"

Reply:
I updated that, thanks